### ISSUE TITLE
Transfer Links to JIRA

### ISSUE THREAD

**Issue by laurafitzgerald** [aerogear/trira#4]
Is it possible to add this feature?

**What**
Pick up links to other cards in trello and transfer that to the JIRA

**Why**
Trello offers the feature to link cards to each other. This is also a feature of JIRA (i.e. tickets which have the status `relates to`)

**Comment by kpiwko** [comment]
This sounds like a complex task. If I understand that correctly, you would like the tool to the following

1/ Create issues in JIRA according to filtered Trello cards
2/ Identify relations in between Trello cards that are in the the filter
3/ Reconstruct the same relation in between JIRA items that have been created

Is that correct?

**Comment by laurafitzgerald** [comment]
@kpiwko yes, you have described it correctly. It's no problem to do this manually so if the task is very complex then feel free to close it. I've been using the tool recently in Planning meetings where we have been using the trello board more as a white board for tasking out stories. Hence the few github issues arising.

**Comment by kpiwko** [comment]
Cool, always happy to improve the tool. The task is not so complex per se, it is rather about possible corner cases.

**Comment by laurafitzgerald** [comment]
Yes this one for us could be considered corner case. Happy to leave it sit here in case it's of interest to others. If not, no hassle ;)
